pkg/locations/start.go

```diff
@@ -37,8 +38,10 @@ func Start(ctx context.Context, name string) error {
 	if err != nil {
 		return fmt.Errorf("unable to update private location on Escape Platform: %w", err)
 	}
-
-	go kube.Start(ctx, id, name, healthy)
+	if os.Getenv("ESCAPE_K8S_INTEGRATION") == "true" {
```

Comment:
here I would say if it's not defined, fallback to true to be consistent with previous versions